### ISSUE TITLE
Accessibility - Using aria-expanded for dropdown

### ISSUE THREAD

**Issue by rohitpaul** [CMSgov/eAPD#2504]
The dropdown on the top right doesn't identify that it has been expanded based once it has been selected.

### This task is done when...
 - [ ] Recommend using `aria-expanded` to notify the user using assisted technologies when the dropdown has been selected and the options have been expanded.

Reference: [w3.org example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html)

**Comment by thetif** [comment]
@rohitpaul which dropdown are you talking about here? could you add a little more context?

**Comment by rohitpaul** [comment]
I'm checking in with my accessibility SME but looks like the tag is already there. Going to clarify if there were any other issues.

**Comment by nicholeweems** [comment]
Per Rohit, this issue has been since tested and is working as expected via Slack on 9/26 and suggests closing the issue.
